src/common/links.ts

```diff
@@ -31,6 +31,7 @@ export const getSourceLink = (
 
 export const scoutArticleLink = `${process.env.COMMENTS_PREFIX}?scout=true`;
 export const squadCreateLink = `${process.env.COMMENTS_PREFIX}?squad=true`;
+export const subscribeNotificationsLink = `${process.env.COMMENTS_PREFIX}/notifications?subscribe=true`;
```

Comment:
This should rather be on the homepage no?

Reply:
Not really sure where the best possible page is, but if it is going to be the notifications page, there would be at least the banner to see where the content changes once you have accepted.